pkg/api/annotation_key_constants.go

```diff
@@ -72,6 +72,10 @@ const (
 	// This annotation can be attached to node.
 	ObjectTTLAnnotationKey string = "node.alpha.kubernetes.io/ttl"
 
+	// CheckpointAnnotationKey represents a Resource (Pod) that should be checkpointed by
+	// the kubelet prior to running
+	CheckpointAnnotationKey string = "node.kubernetes.io/checkpoint"
```

Comment:
`alpha.node.kubernetes.io/bootstrapping-checkpointing`?
`node.alpha.kubernetes.io/bootstrapping-checkpointing`?

I guess that when this gets graduated we'll use an alpha/beta/whatever field using the grand new proposal we have for that, right?

Reply:
I'm keeping the annotation name the same for the purpose of potential re-use.  The documentation update will call out the details and limitations.